docs/maps/import-geospatial-data.asciidoc

```diff
@@ -6,6 +6,30 @@ To import geospatical data into the Elastic Stack, the data must be indexed as {
 Geospatial data comes in many formats.
 Choose an import tool based on the format of your geospatial data.
 
+[discrete]
+[[import-geospatial-privileges]]
+=== Security privileges
+
+The {stack-security-features} provide roles and privileges that control which users can upload files.
+You can manage your roles, privileges, and
+spaces in the **{stack-manage-app}** app in {kib}. For more information, see
+{ref}/security-privileges.html[Security privileges],
+<<kibana-privileges, {kib} privileges]>>, and <<xpack-kibana-role-management, {kib} role management>>.
+
+To upload GeoJson files in {kib} with *Maps*, you must have:
+* [ ] `all` {kib}
+privilege for the `Maps` feature
+* [ ] `all` {kib}
+privilege for the `Index Pattern Management` feature
```

Comment:
```suggestion
privilege for the *Index Patterns* feature in *Stack Management*
```

Reply:
Need to keep the word "Index Pattern Management", https://www.elastic.co/guide/en/kibana/current/index-patterns.html#index-patterns-read-only-access. That is the name of the feature priviliage in the UI

<img width="400" alt="Screen Shot 2021-03-31 at 4 03 52 PM" src="https://user-images.githubusercontent.com/373691/113217151-b6f1f200-923a-11eb-9806-326ff178159a.png">